test/test_with_xpress.jl

```diff
@@ -1377,6 +1377,82 @@ end
     @test calculated_ghp_capex ≈ reopt_ghp_capex atol=300
 end
 
+@testset "Cambium Emissions" begin
```

Comment:
Is this the same test as in "runtests.jl" with the HiGHS solver? If so, we don't need to add it here. We should be prioritizing adding tests to runtests.jl with an open source solver whenever possible.

Reply:
yes, this was exactly the same as the one in runtests. I deleted this testset in `test_with_xpress` in https://github.com/NREL/REopt.jl/pull/357/commits/cb6d9d9da27e0dfc26ad5c1cbceea9ac0f3a88f1